### PR TITLE
Add subsection about required packages

### PR DESCRIPTION
**Pull request by gassmoeller**
This is a first version that tries to address #324. Let me know about your opinion. We can easily add commands at the end for different package managers or extend it to subsubsections for different operating systems.

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -1568,7 +1568,41 @@ \section{Installation}
...
 \aspect{} itself.
 
-\subsection{Prerequisites}
+\subsection{System prerequisites}
+
+In order to install \aspect{} and its dependencies, you should
+have your system set up to be able to compile and link programs. Additionally,
+\aspect{} needs a number of widely used libraries that are available
+for most operating systems. Therefore, you will need compilers for C, C++ and
+Fortran, the GNU make system, the CMake build system, development versions of
```
Comment (bangerth) [inline doc/manual/manual.tex]: do we really need fortran?

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -1568,7 +1568,41 @@ \section{Installation}
...
+the BLAS and LAPACK libraries and the zlib library, which is used for
+compressing the output data. To use more than one process for your computations
+you will need to install a development version of a MPI library and the
```
Comment (bangerth) [inline doc/manual/manual.tex]: To me, "development versions" always mean that they include the actual source code. Some package managers run them as "openmpi-dev", for example. But that's not really what you need. Rephrase?

File: doc/manual/manual.tex [hunk 3]
```diff
@@ -1568,7 +1568,41 @@ \section{Installation}
...
+necessary executables to run MPI programs. There are some optional packages
+for additional features, like the HDF5 libraries for additional output formats
+and PETSC for alternative solvers, but these are not strictly required.
+Numdiff is used for checking \aspect{}'s test results with reasonable accuracy.
```
Comment (bangerth) [inline doc/manual/manual.tex]: Might be worth pointing out that PETSc and numdiff may not be available through the package manager of all systems. (I suppose so, at least, but can't really say.)

File: doc/manual/manual.tex [hunk 4]
```diff
@@ -1568,7 +1568,41 @@ \section{Installation}
...
+Finally, for obtaining a recent development version of \aspect{} you will
+need the git version control system.
+
+An exemplary command to obtain all required packages on Ubuntu 14.04 would be:
+
+\begin{verbatim}
```
Comment (bangerth) [inline doc/manual/manual.tex]: drop the empty line.

**Comment by bangerth** [comment]
Nice. Please address comments as you see fit and merge yourself.

File: doc/manual/manual.tex (hunk 1 above)
Comment (gassmoeller) [inline doc/manual/manual.tex]: Without fortran the cmake configuration of trilinos-11.8.1 fails on my system ("The fortran compiler "/usr/bin/mpif90" was not able to compile a simple test program"). Maybe it is related to what MPI compilers are delivered in the MPI package? Anyway, it seems like on Ubuntu this is necessary.

File: doc/manual/manual.tex (hunk 2 above)
Comment (gassmoeller) [inline doc/manual/manual.tex]: This seems to be distribution dependent. On Ubuntu the normal packages only include the shared libraries, the -dev packages additionally include the header files (which we need), and there are separate packages that also include the source files (on normal system without changes these are not even visible). But I can clarify that we need the libraries and headers, and on Ubuntu this means installing the -dev versions of the packages.

File: doc/manual/manual.tex (hunk 1 above)
Comment (bangerth) [inline doc/manual/manual.tex]: OK, that's fine then.

File: doc/manual/manual.tex (hunk 2 above)
Comment (bangerth) [inline doc/manual/manual.tex]: Good point.

**Pull request merged**